src/GRA.Controllers/EventsController.cs

```diff
@@ -9,25 +10,38 @@
 using GRA.Domain.Model.Filters;
 using GRA.Domain.Service;
 using GRA.Domain.Service.Models;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.Extensions.Logging;
 
 namespace GRA.Controllers
 {
     public class EventsController : Base.UserController
     {
+        public const string VisitedAll = "All";
```

Comment:
What are these constants? Do they need to be internationalized?

Reply:
Part of @iafb's changes

Reply:
Those constants are for the url routing parameter values, I don't think they need to be internationalized.